batch/scripts/get_arguments.sh

```diff
@@ -172,6 +177,11 @@ do
       SKIP="TRUE"
       shift # past argument
       ;;
+      --srid)
+      SRID="$2"
+      shift # past argument
+      shift # past value
+      ;;
```

Comment:
Was this new --srid ever used?  I don't see it used in cogify.sh

Reply:
We ended up removing it, good catch!